pkg/util/crypt/crypt_dev.go

```diff
@@ -169,7 +174,7 @@ func (crypt *Device) FormatCryptDevice(path, key string) (string, string, error)
 	defer lock.Release(fd)
 
 	nextCrypt := getNextAvailableCryptDevice()
-	cmd = exec.Command("/sbin/cryptsetup", "luksOpen", loopdev, nextCrypt)
+	cmd = exec.Command(cryptsetup, "luksOpen", loopdev, nextCrypt)
 	cmd.Dir = "/dev"
```

Comment:
Running cryptsetup in "/dev" is harmless in this case I feel. But, as you noted in slack, you can delete this and pass "/dev"+loopdev I guess

Reply:
I'll fix that in a different PR.